### ISSUE TITLE
Simple setup prompt on first run

### ISSUE THREAD

**Issue by yuzi-co** [fireice-uk/xmr-stak#2382]
With simple setup added, with pool parameters provided, there is a prompt for the simple configuration and it continues.

Previously it would configure hardware automatically and start mining.

This prevents auto mining scripts when starting mining with a new version, or requires to manually create configs before run.

A command line flag would be useful in this case, or even just skip the prompt since the needed options are already provided.

Example command line:
```
xmr-stak -o cryptonightheavy.eu.nicehash.com:3364 -u xxx -p x --noNVIDIA --noAMD --noUAC --use-nicehash --httpd 4048 --cpu CPU-CnHeavy.txt --currency cryptonight_heavy
```

**Comment by Spudz76** [comment]
You missed `--rigid`

[See code](https://github.com/fireice-uk/xmr-stak/pull/2363/files/ac03aba24a9d39aa0b3d4c952dae9af5b3079e5d#diff-6c13e8fdffaf680d2e2876486b6124abR260) wherein if that's empty, you get simple setup'd.

**Comment by yuzi-co** [comment]
same:
```
./xmr-stak -o cryptonightheavy.eu.nicehash.com:3364 -u xxx -p x --rigid xxx --noNVIDIA --noAMD --noUAC --use-nicehash --httpd 4048 --cpu CPU-CnHeavy.txt --currency cryptonight_heavy

Use simple setup method? (Y/n)
```

**Comment by psychocrypt** [comment]
Ohh yes could be that this is a bug. I only check if the files exists but I
need to check also if rhe user passed command line options.

I will check it.

**Comment by vanities** [comment]
A command line argument would be wonderful, I run xmr-stak inside of a container and I'm having to do some weird `(no | ./xmr-stak) | yes` in my Dockerfile in order. for it to run as an instance. :(

**Comment by psychocrypt** [comment]
solved in #2385
